docs/getting_started/index.rst

```diff
@@ -490,6 +502,60 @@ Yielding the output:
    :lines: 1-12
 
 
+Building SPDK backend on Windows
+--------------------------------
+
+**SPDK** can be used as a backend for **xNVMe**, to leverage this interface
+first user need to compile the **SPDK** as a subproject with **xNVMe** its
+depend on Windows Platform Development Kit (WPDK) that help to enables
+applications based on the Storage Performance Development Kit (SPDK) to
+build and run as native Windows executables.
```

Comment:
This paragraph is unclear to me. It reads like SPDK depends on WPDK, is that true?
Perhaps it can be simplified.